### PR TITLE
Functionality for Customer Password

### PR DESCRIPTION
**Pull request by SamTakhar12**
Modified NewBank
Modified ExampleClient
Added Customer Password class

### CODE REVIEW

**Comment by carolinamartes** [comment]
Could we remove all the xml and iml autogenerated files from here? Don't think we need to push them into the repo

File: newbank/client/ExampleClient.java [hunk 1]
```diff
@@ -8,50 +8,55 @@
...
-			} catch (IOException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
-			}
-		}
-	}
-	
-	public static void main(String[] args) throws UnknownHostException, IOException, InterruptedException {
-		new ExampleClient("localhost",14002).start();
-	}
```
Comment (carolinamartes) [inline newbank/client/ExampleClient.java:56]: these changes are all just indentation, right?

Comment (SamTakhar12) [reply newbank/client/ExampleClient.java:56]: Yes, looks like changes to the indentation. I did also have a method at Line 46:
 if(command.equals("LOGOUT")){
                        bankServerOut.close();
                        server.close();
                        System.exit(0);
                    }

Which I can't see in this pull request.

**Comment by carolinamartes** [comment]
@SamTakhar12 let me know when you've resolved the conflicts and deleted the extra files if you agree and I can approve to merge 👍

**Comment by carolinamartes** [comment]
Also @SamTakhar12 Did you push those changes re Logout to [this PR](https://github.com/carolinamartes/new_bank_group3/pull/1/files)?

**Comment by mileseu** [comment]
Replaced with #11

**Pull request closed without merging**